sdk/search/Microsoft.Azure.Search/tests/SessionRecords/EncryptionWithCustomerManagedKeysTests/CreateIndexWithCustomerManagedKeysUsingAccessCredentials.json

```diff
@@ -7,48 +7,48 @@
       "RequestBody": "",
       "RequestHeaders": {
         "x-ms-client-request-id": [
-          "d9f26adf-bdc6-4251-a0ee-cf3e57bfdb7d"
+          "458eff35-1b96-411c-adb3-f112e2ef3dfc"
         ],
-        "accept-language": [
```

Comment:
Any particular reason for the casing to change here?

Reply:
None, but I think it doesn't matter for http headers... This is auto-generated by the test tool, so I didn't hand edit it